### ISSUE TITLE
Build error when deploying via Now - unable to use styled-components

### ISSUE THREAD

**Issue by tedjames** [zeit/next.js#4085]
I'm interested in using styled-components with NextJS SSR and deploying via Now. While everything builds and runs without errors locally (as shown in the image below), attempting to deploy via Now results in one of two errors as shown below in the current behavior section.

## Expected Behavior

The app should deploy via Now successfully and render index.js with styled-components just as it does when running `npm run dev` or `npm run start`.

This is what the rendered page should look like:
<img width="398" alt="screenshot at mar 30 14-00-54" src="https://user-images.githubusercontent.com/13091675/38148065-d5c55290-3422-11e8-9f13-bb034ba5faa5.png">

This is the expected response from now-cli:
![screenshot at mar 30 14-10-55](https://user-images.githubusercontent.com/13091675/38148377-445d7308-3424-11e8-9ad1-de54f90fbd45.png)

## Current Behavior
Error 1 (no longer appearing for some reason): 
![screenshot at mar 30 14-05-05](https://user-images.githubusercontent.com/13091675/38148244-9dde15c8-3423-11e8-9a69-c10736322427.png)
![screenshot at mar 30 14-06-03](https://user-images.githubusercontent.com/13091675/38148247-a25b2820-3423-11e8-9995-6035570519c2.png)

Error 2 (this one is now constantly appearing):
![screenshot at mar 30 14-08-10](https://user-images.githubusercontent.com/13091675/38148305-e2a1a774-3423-11e8-8c26-40b4cb285971.png)

## Steps to Reproduce (for bugs)
1. Download and install Now - [https://zeit.co/now#get-started](url)
2. Clone my repo: `git clone https://github.com/tedjames/nextjs-deploy.git`
3. Run: `npm install`
4. Run: `npm run build`
5. Run: `now`

## Context
- The application can only be deployed via Now by removing the styled-components package and removing all instances of styled-components from `pages/index.js` and `pages/_document.js`
- I'm using this tutorial for configuring styled-components: [https://bilalbudhani.com/using-next-js-with-styled-components/](url)

## Your Environment

| Tech    | Version |
|---------|---------|
| npm     |    5.8.0     |
| yarn    |    1.5.1     |
| node    |    8.9.4     |
| next    |    ^5.1.0     |
| OS      |    Mac OS Sierra 10.12.6 (16G1114)     |
| browser |    Google Chrome     |
| express     |    ^4.16.3     |
| react     |    ^16.3.0     |
| styled-components     |    ^3.2.5     |

Any help would be greatly appreciated! 

I'm really excited about using NextJS but without knowing why a simple app like this can't be deployed via Now, I don't feel comfortable using this framework on some upcoming projects that could really benefit from SSR.

Thanks,

Ted

**Comment by tedjames** [comment]
Thanks to @timneutkens, this issue has been solved 🎉 

I had somehow ended up with both a `package-lock.json` and `yarn.lock` which caused the error.
After deleting both those files and running `yarn` to regenerate the lock file, I was able to finally deploy via Now.

On another note, the lack of an error message is apparently a separate issue itself which @timneutkens is now in the process of resolving. I'll leave this issue open for now until this has been resolved or until a separate issue has been opened.
